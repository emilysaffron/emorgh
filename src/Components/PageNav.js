import CalculatePageNumber from "../helpers/CalculatePageNumber";
import { Link } from "@reach/router";
import { NavigationLi } from "@bbc/psammead-navigation";
const PageNav = ({ currentPageNumber, onClick, direction }) => {
  const pageNumber = CalculatePageNumber(currentPageNumber, direction);

  return (
    <Link to={`/article${pageNumber}`}>
      <NavigationLi onClick={onClick}>{direction}</NavigationLi>
    </Link>
  );
};

export default PageNav;
