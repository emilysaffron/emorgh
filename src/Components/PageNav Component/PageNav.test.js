import PageNav from "./PageNav";
import renderer from "react-test-renderer";

it("PageNav Snapshot", () => {
  const tree = renderer.create(<PageNav />).toJSON();
  expect(tree).toMatchSnapshot();
});
