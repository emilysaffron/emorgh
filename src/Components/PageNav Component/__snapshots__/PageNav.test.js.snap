// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNav Snapshot 1`] = `
<a
  href="/articleundefined"
  onClick={[Function]}
>
  <li
    className="css-1vectme-StyledListItem e1lim4kn3"
    dir="ltr"
    role="listitem"
  >
    <a
      className="css-cig9wk-StyledLink e1lim4kn2"
    />
  </li>
</a>
`;
